testdata/source/schwab_csv/test_basic/transactions/EAC_Checking_Transactions_20201223-160009.CSV

```diff
@@ -0,0 +1,12 @@
+Transactions  for Checking account EAC as of 12/23/2020 16:00:10 ET
+"Date","Type","Check #","Description","Withdrawal (-)","Deposit (+)","RunningBalance"
+"Pending Transactions are not reflected within this sort criterion."
+"Posted Transactions""11/13/2020","TRANSFER","","Funds Transfer to Brokerage -9999","$10,500.00","","$50.01"

```

Comment:
Something looks mis-formatted here. Is that coming directly from Schwab, or was accidentally introduced in this test file?
 
If the latter, let's re-verify the test file looks just like the CSV files you get from upstream.

If the former, that's unfortunate, but the importer needs adjusting to account for it so we don't skip the first transaction in every CSV. This row is currently silently ignored.